src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -752,11 +752,17 @@ void regclass_graph_Model(py::module m) {
     model.def(
         "has_rt_info",
         [](const ov::Model& self, const py::list& path) -> bool {
-            std::vector<std::string> cpp_args(path.size());
-            for (size_t i = 0; i < path.size(); i++) {
-                cpp_args[i] = path[i].cast<std::string>();
+            // FIXME: understand why has_rt_info causes Python crash
+            try {
+                std::vector<std::string> cpp_args(path.size());
+                for (size_t i = 0; i < path.size(); i++) {
+                    cpp_args[i] = path[i].cast<std::string>();
+                }
+                self.get_rt_info<ov::Any>(cpp_args);
+                return true;
+            } catch (ov::Exception&) {
+                return false;
```

Comment:
who will investigate this?

CC @jiwaszki

Reply:
I created ticket for Python team

Reply:
we have an issue connected with RTMap -- #98169